styles.css

```diff
@@ -0,0 +1,133 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+.main {
+  background: url("images/IMG-20220819-WA0005.jpg");
+  background-repeat: no-repeat;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 100%;
+  font-family: Poppins, Arial, Verdana, sans-serif;
+}
+
+.top {
+  display: flex;
+  height: auto;
+  flex-direction: column;
+  margin-top: -15px;
+  background-size: auto;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.header {
+  margin: auto;
+  position: center;
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  font-family: Poppins, Arial, Verdana, sans-serif;
+}
```

Comment:
- [ ] Please make your header transparent and remove the underline under your logo.